### PR TITLE
Dpn-33/solenoid diagram

### PR DESCRIPTION
**Pull request by trevorliberty**
Made a lot of index.html into components
Added diagrams
Added tooltips

### CODE REVIEW

**Review by sherretten — APPROVED** [review]
This looks great. Thanks for moving the graph HTML to its own template. It now makes a lot of sense that it's done that way!

File: static/js/main.js [hunk 1]
```diff
@@ -183,6 +203,9 @@ const createInputs = () => {
       }
     }
   });
+  $(function () {
```
Comment (t-john93) [inline static/js/main.js:206]: Nitpicky but can you change this to es2015 fat arrow syntax?

Comment (t-john93) [reply static/js/main.js:206]: Other than than LGTM.

File: static/js/main.js [hunk 2]
```diff
@@ -38,56 +38,64 @@ const createInputs = () => {
     {
       name: SolenoidParameters.VOLTAGE,
       symbol: 'V',
-      value: urlParams.get(SolenoidParameters.VOLTAGE) || '',
+      value: urlParams.get('voltage') || '',
```
Comment (jjesse55) [inline static/js/main.js:41]: Just curious, is there a reason you decided to use the literal strings instead? Everything else LGTM

Comment (trevorliberty) [reply static/js/main.js:41]: Nah, just thought it was cleaner

**Review by jjesse55 — APPROVED** [review]

**Pull request merged**